.eslintrc.json

```diff
@@ -1,7 +1,19 @@
 {
     "env": {
         "browser": true,
-        "es2021": true
+        "es2021": true,
+        "node": true,
+        "jest": true
+    },
+    "globals": {
+        "click": "readonly",
+        "createAsyncSpec": "readonly",
+        "assert": "readonly",
+        "cleanTestPage": "readonly",
+        "waitForStyles": "readonly",
+        "interactionManager": "readonly",
+        "mockEventObject": "readonly",
+        "dragIMElement": "readonly"
```

Comment:
Can we try and add the config options for the tests in the tests package.json like this: 
```
"eslintConfig": {
    "globals": {
        
    },
    "env": {
      "jest": true
    }
}
```

IMO it will make the config more readable if everything is separate, but you can decide if it's worth it.